libs/blocks/faas/faas.js

```diff
@@ -10,4 +12,16 @@ export default async function init(a) {
   if (faas && faas.closest('.dialog-modal')) {
     faas.querySelector('.faas').classList.add('column2');
   }
+};
+
+export default async function init(a) {
+  if (a.textContent.includes('no-lazy')) {
+    loadFaas(a);
+  } else {
+    createIntersectionObserver({
+      el: a,
+      options: { rootMargin: `${ROOT_MARGIN}px` },
```

Comment:
super nit - may as well set `const ROOT_MARGIN = '1000px';`

Reply:
It is already set at line 4.